app/main.py

```diff
@@ -1,3 +1,56 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.customer import Customer
+from app.shop import get_shop_list
+from app.car import Car
+
+
+def get_customer_list(data: dict) -> list[Customer]:
+    customers_list = data.get("customers")
+    fuel_price = data.get("FUEL_PRICE")
+
+    return [
+        Customer(
+            name=person.get("name"),
+            product_cart=person.get("product_cart"),
+            location=person.get("location"),
+            money=person.get("money"),
+            car=Car(
+                person["car"].get("brand"),
+                person["car"].get("fuel_consumption"),
+                fuel_price
```

Comment:
If you'll put `self.car = Car(**car_data)` in Customer init, you could use unpacking also here